README.md

```diff
@@ -69,13 +69,28 @@ Junrar.extract(resourceAsStream, tempFolder);
 ```java
 // Assuming you already have an InputStream from the rar file and an OutputStream for writing to
 final Archive archive = new Archive(inputStream);
-  while (true) {
-      FileHeader fileHeader = archive.nextFileHeader();
-      if (fileHeader == null) {
-          break;
-      }
-      archive.extractFile(fileHeader, outputStream); 
+while (true) {
+  FileHeader fileHeader = archive.nextFileHeader();
+  if (fileHeader == null) {
+      break;
   }
+  archive.extractFile(fileHeader, outputStream); 
+}
+```
+
+### Extract from an InputStream to an InputStream
+```java
+// Assuming you already have an InputStream from the rar file and an OutputStream for writing to
+final Archive archive = new Archive(inputStream);
+while (true) {
+  FileHeader fileHeader = archive.nextFileHeader();
+  if (fileHeader == null) {
+      break;
+  }
+  InputStream is = archive.getInputStream(fileHeader);
```

Comment:
It would be nice to add a try-with-resources statement here in the example. People who just copy the example as is could potentially introduce a resource leak here.

Reply:
Completely forgot about this. Gonna do it later today or tomorrow.